src/CDC2021/tests/test_articulated_vehicle.jl

```diff
@@ -0,0 +1,395 @@
+include(joinpath("..", "..", "Abstraction", "abstraction.jl"))
+include("../general_domain.jl")
+include("../utils.jl")
+include("../alternating_simulation.jl")
+include("../partition.jl")
+include("../lazy_abstraction.jl")
+include("../branch_and_bound.jl")
+include("../optimal_control.jl")
```

Comment:
I think these should be accessed through Dionysos package and not directly through the files like this...